parking_permits/services/kmo.py

```diff
@@ -71,11 +81,30 @@ def parse_street_name_and_number(street_address):
     )
 
 
-def get_address_detail_from_kmo(street_name, street_number):
+def get_address_detail_from_db(street_name, street_number):
+    address_qs = Address.objects.filter(
+        street_name=street_name, street_number=street_number
+    )
+    if address_qs.exists():
+        return address_qs.first()
+
+
+def get_address_details(street_name, street_number):
     results = get_wfs_result(street_name, street_number)
```

Comment:
Isn't this suppose to be other way round. First query our database and if not found query KMO. IMHO we should try to avoid external API call as much as possible.